onboard/nav/stateMachine.cpp

```diff
@@ -405,6 +493,25 @@ bool StateMachine::isWaypointReachable( double distance )
     return isLocationReachable( mPhoebe, mRoverConfig, distance, mRoverConfig["navThresholds"]["waypointDistance"].GetDouble());
 } // isWaypointReachable
 
+bool StateMachine::isRadioRepeaterDrop() const
```

Comment:
comment function. I'm also confused by this naming. Is it checking something or performing an action?